pages/index.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+	<main>
+		<navbar />
```

Comment:
For the `NuxtLink` you used CamelCase but now dont. Should stick to it

Reply:
That use of `<navbar />` has been removed, uses CamelCase in `layouts/default.vue`